### ISSUE TITLE
How to use this in project Xamarin.Forms Portable?

### ISSUE THREAD

**Issue by yauhenipakala** [Microsoft/ApplicationInsights-Xamarin#10]
When you try to install the Nuget package, I get the error:

```
Could not install package 'Microsoft.ApplicationInsights.Xamarin 1.0.0-alpha2'.
You are trying to install this package into a project that targets '.NETPortable,Version=v4.5,Profile=Profile78',
but the package does not contain any assembly references or content files that are compatible with that framework.
```

Can I use ApplicationInsights in a Portable project?

**Comment by ghost** [comment]
Where did you get the nupkg from (which branch)?

**Comment by yauhenipakala** [comment]
From `master` branch

**Comment by yauhenipakala** [comment]
If I use the nupkg from [feature/change-pcl-profile](https://github.com/Microsoft/ApplicationInsights-Xamarin/tree/feature/change-pcl-profile) get the error:

```
Failed to add reference to 'AI.XamarinSDK.Abstractions'.
```

I use Visual Studio 2015.
My test project can be seen here: https://github.com/wcoder/AppInsightsForms (now use custom wrapper class)

**Comment by ghost** [comment]
Hi,

just try to remove the packages from both the shared project as well as the Android project and add them again:

![screen shot 2015-09-29 at 11 30 53](https://cloud.githubusercontent.com/assets/10158882/10174336/2f38912a-669e-11e5-9db8-65a0e7b08a42.png)

Thank you,
Chris

**Comment by yauhenipakala** [comment]
Hi,

I tried this way, if I install package with Xamarin Studio, get the error:

```
Adding Microsoft.ApplicationInsights.Xamarin...
Attempting to resolve dependency 'Xamarin.Forms (≥ 1.4.3.6374)'.
Adding 'Microsoft.ApplicationInsights.Xamarin 1.0.0-alpha2' to AppInsightsForms.

Could not install package 'Microsoft.ApplicationInsights.Xamarin 1.0.0-alpha2'.
You are trying to install this package into a project that targets
'portable-net45+dnxcore50+win+wp80+MonoAndroid10+xamarinios10+MonoTouch10',
but the package does not contain any assembly
references or content files that are compatible with that framework.
For more information, contact the package author.
```

**Comment by ghost** [comment]
Which Xamarin versions are you using?

**Comment by yauhenipakala** [comment]
I use:

```
Microsoft Visual Studio Enterprise 2015 Version 14.0.23107.0 D14REL

NuGet Package Manager   3.2.0

Xamarin   3.11.1443.0
Visual Studio extension to enable development for Xamarin.iOS and Xamarin.Android.

Xamarin.Android   5.1.6.7
Visual Studio plugin to enable development for Xamarin.Android.

Xamarin.iOS   9.0.1.0
Visual Studio extension to enable development for Xamarin.iOS.

Xamarin.Forms Intellisense   1.0
Provides intellisense for Xamarin.Forms in the XML editor.
```

Info from `VS -> Help -> About`

**Comment by ghost** [comment]
Hi,

thanks for the info. It seems like some of your app assemblies don't support profile 78. I'm going to have a second look at it on my Windows machine. In the meanwhile, can you please compare your app config with the one of the demo app which comes with the sdk?

Best,

-Christoph

**Comment by ghost** [comment]
Hi,

Are you sure we are talking about the same [demo project](https://github.com/wcoder/AppInsightsForms)? I just downloaded the project on my windows machine and opened it with Xamarin Studio. Then I removed the AI Xamarin SDK NuGet from the Droid target and added it again – without any problem. I repeated this step in VS 2015 and it worked as well. Did you modify your local version in the meanwhile? Can you assist me to be able to reproduce the behaviour? Are you still using the package from the feature/change-pcl-profile branch (now develop)?

Thank you

**Comment by yauhenipakala** [comment]
Hi Chris,

I solved this problem. It is my local problem!

The problem occurs because:
- I downloaded zip archive from `develop` branch
- unarchiving it I got the following path to the package: `C:\Users\wcoder\Desktop\ApplicationInsights-Xamarin-develop\ApplicationInsights-Xamarin-develop\ApplicationInsightsXamarinSDK\NuGet`
- this path too long, so failed to add package to my project :smile: 

Thank you for your time.
